src/main/app.js

```diff
@@ -59,37 +68,96 @@ fluid.defaults("gpii.app", {
             activeSet: null
         }
     },
-    // prerequisites
+    members: {
+        machineId: "@expand:{that}.installID.getMachineID()"
+    },
     components: {
-        psp: {
-            type: "gpii.app.psp",
-            createOnEvent: "onPrerequisitesReady",
+        networkCheck: { // Network check component to meet GPII-2349
+            type: "gpii.app.networkCheck"
+        },
+        installID: {
+            type: "gpii.installID",
+            priority: "after:networkCheck"
+        },
+        factsManager: {
+            type: "gpii.app.factsManager",
+            priority: "after:installID"
+        },
+        rulesEngine: {
+            type: "gpii.app.rulesEngine",
+            priority: "after:factsManager",
```

Comment:
For example, this annotation is unnecessary since the options for the component contain an explicit reference to the factsManager which guarantees that it will be available